vars/packageBuildingPipelineDAOSTest.groovy

```diff
@@ -755,8 +755,6 @@ void call(Map pipeline_args) {
                                                    booleanParam(name: 'CI_SCAN_RPMS_el7_TEST', value: false),
                                                    booleanParam(name: 'CI_RPMS_el7_TEST',
                                                                 value: 'centos7' in distros),
-                                                   booleanParam(name: 'CI_small_TEST',
-                                                                value: 'el8' in distros),
```

Comment:
This is removing all hardware testing from dependency packages.  We should run at least one hardware stage on them.  It seems it should be medium now though.

Reply:
Fixed